cmatrix.spec

```diff
@@ -0,0 +1,46 @@
+%define name    cmatrix
```

Comment:
This file is dynamically generated from .spec.in file using auto tools for distribution as an rpm file. Please don't include it.